spec/models/api/v1_0/degree_attributes_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Api::V10::DegreeAttributes do
```

Comment:
Not too sure with duplicating the tests seems too much, causing it to be fragile...

Maybe we should extract it so we have some reuse value, a shared example or something.

If we get to x version and nothing was changed then we have x copy of the same lines of tests.